tests/acceptance/run.sh

```diff
@@ -82,7 +82,7 @@ if [ -n "${EXPECTED_FAILURES_FILE}" ]; then
   # Check that every failed scenario is in the list of expected failures
   # Loop through the keys of the FAILED_SCENARIO_PATHS array (! does that)
   for FAILED_SCENARIO_PATH in "${!FAILED_SCENARIO_PATHS[@]}"; do
-    grep -x "${FAILED_SCENARIO_PATH}" "${EXPECTED_FAILURES_FILE}" >/dev/null
+    grep -v "${FAILED_SCENARIO_PATH}" "${EXPECTED_FAILURES_FILE}" >/dev/null
```

Comment:
I think that we will need to search for the string and the brackets around it, `[someSuite/login.feature:42]`
If we just grep for `someSuite/login.feature:42` then it will also match `[someSuite/login.feature:420]` `[someSuite/login.feature:421]` etc.